e2e/testdata/fn-eval/missing-fn-config/.expected/config.yaml

```diff
@@ -18,4 +18,3 @@ image: gcr.io/kpt-fn/set-namespace:v0.1
 stdErr: "failed to configure function: input namespace cannot be empty"
 stdOut: |
   [RUNNING] "gcr.io/kpt-fn/set-namespace:v0.1"
-  [FAIL] "gcr.io/kpt-fn/set-namespace:v0.1"
```

Comment:
Why this change?

Also, I've commented on this in couple of other PRs. What's the rationale for what's printed to stderr vs stdout? Why not just use stderr like when piping? Also if you print to different streams, then they may be presented out of order.

Reply:
> Why this change?

I had a discussion with @droot and we agreed that the line `[FAIL] ...` is an error message and should be printed to stderr to provide enough information about which function fails in a command pipe.

> Also, I've commented on this in couple of other PRs. What's the rationale for what's printed to stderr vs stdout? Why not just use stderr like when piping? Also if you print to different streams, then they may be presented out of order.

The detail design for what should be printed to stderr and what should be printed to stdout should be done in #1915. We can have tests to cover the behavior before #1915 is done.

Reply:
ok. I think using two channels is not the way to go for the reasons I stated. So consider it designed. Won't block this PR.